mono/mini/tramp-wasm.c

```diff
@@ -7,7 +7,7 @@ void mono_sdb_single_step_trampoline (void);
 gpointer
 mono_arch_create_specific_trampoline (gpointer arg1, MonoTrampolineType tramp_type, MonoDomain *domain, guint32 *code_len)
 {
-	//FUN FACT but this is a key used to do reverse lookups when resolving patch entries!
+	// FIXME? Historically FUN FACT but this is a key used to do reverse lookups when resolving patch entries!
```

Comment:
This can probably return the same address for every call now.

Reply:
Modified to return one function that does g_error.
Previously was returning data for a code pointer, so..